spec/liquid_tags/youtube_tag_spec.rb

```diff
@@ -36,36 +36,31 @@ def generate_new_liquid(id)
       Liquid::Template.parse("{% youtube #{id} %}")
     end
 
-    def generate_iframe(id)
-      "<iframe "\
-        "width=\"710\" "\
-        "height=\"399\" "\
-        "src=\"https://www.youtube.com/embed/#{parsed_id(id)}\" "\
-        "allowfullscreen> "\
-      "</iframe>"
-    end
-
     it "accepts a valid YouTube ID with no starting time" do
-      liquid = generate_new_liquid(valid_id_no_time)
-      expect(liquid.render).to eq(generate_iframe(valid_id_no_time))
+      liquid = generate_new_liquid(valid_id_no_time).render
+      Approvals.verify(liquid, name: "youtube_liquid_tag_no_time", format: :html)
     end
 
     it "accepts valid YouTube IDs with starting times" do
+      count = 0
```

Comment:
Generally, it's bad practice to use a count outside of a loop. Instead, use `each_with_index` to clean it up.
```ruby
valid_ids_with_time.each_with_index do |id, index|
  liquid = generate_new_liquid(id).render
  Approvals.verify(liquid, name: "youtube_liquid_tag_with_time_#{index + 1}", format: :html)
end
```

Reply:
Yeah that's a good suggestion.

Also, you may not need to test all of them. Testing one of them is enough for this case.

Reply:
Think I'll remove the loop and reduce the approval tests since it creates a lot of files.